lib/help/help.go

```diff
@@ -70,12 +69,23 @@ func Separator() string {
 func Separators(os string) string {
 	switch os {
 	case "windows":
-		return "//"
+		return "\\"
 	default:
+		fallthrough
+	case "unix":
 		return "/"
 	}
 }
 
+func AddPathSuffix(path, suffix, os string) string {
```

Comment:
__[golint]__
* [golint] exported function AddPathSuffix should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->